From 4ecf79144184c6736c1eea19fdcedf9cab1e9010 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 18 Feb 2022 07:40:15 +0200
Subject: [PATCH] [SW-0] gaudi3: correct function arguments

This patch correct arguments passed to function
security_report_img_status.

Change-Id: If542a9eb4991bea10977c0970c38b6901f645b84
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index ad11e26..6475a59 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -304,13 +304,13 @@
 			pub_key, desc->pub_key_size, signature, desc->sig_size, desc->sec_ver);
 	if (ret != SECUREBOOT_VERIFY_SUCCESS) {
 		if (ret == SECUREBOOT_VERIFY_VER_FAILED)
-			security_report_img_status(IMG_STATUS_FW_VERSION_LOW, cur_img);
+			security_report_img_status(curr_img, IMG_STATUS_FW_VERSION_LOW);
 		else if (ret == SECUREBOOT_VERIFY_KEY_REVOKED)
 			arcboot_rom_status_update_bringup(img_sts_base, PROGRESS_BRINGUP_KEY_REVOKED);
 		else if (ret == SECUREBOOT_VERIFY_KEY_FAILED)
 			arcboot_rom_status_update_bringup(img_sts_base, PROGRESS_BRINGUP_KEY_AUTH_FAIL);
 		else
-			security_report_img_status(IMG_STATUS_SIG_AUTH_FAIL, cur_img);
+			security_report_img_status(curr_img, IMG_STATUS_SIG_AUTH_FAIL);
 		return ARCBOOT_BRINGUP_ERROR;
 	}
 
